### DISCUSSION TITLE
🔦 Feature:  nocodb packages/nocodb/src/schema/swagger.json Auto generation or manual？

### DISCUSSION THREAD

**Discussion by xinre** [nocodb/nocodb#8258] [Q&A]
### Please confirm if feature request does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

hello  ， I'm looking at your code, and I have a question about whether packages/nocodb/src/schema/swagger.json is being generated automatically or written manually. If it is written manually, can it be generated automatically in the future?

### Suggested Solution

may be openapi-generator swagger etc

**Comment by ZoeLeee** [comment]
Currently, it needs to be maintained manually. The 'api.ts' is generated based on this JSON file
